content/_data/changelogs/weekly.yml

```diff
@@ -1289,6 +1289,69 @@
     # pull: 3055 too minor (test fix)
     # pull: 3063 too minor (test fix)
     # pull: 3070 too minor (suppress test)
+- version: "2.86"
+  date: 2017-10-22
+  changes:
+    - type: major rfe
+      pull: 3076
+      message: >
+        <em>Launch agent via execution of command on the master</em> has been moved to a new <em>Command Launcher</em> plugin and integrated with the <em>Script Security</em> plugin.
+      references:
+        - issue: 47393
+        - url: https://jenkins.io/security/advisory/2017-10-11/#arbitrary-shell-command-execution-on-master-by-users-with-agent-related-permissions
+          title: related security advisory
+    - type: rfe
+      issue: 36282
+      pull: 3081
+      message: >
+        Add link to recursive <code>cc.xml</code> output on build history page.
+    - type: major bug
+      issue: 47448
+      pull: 3093
+      message: >
+        Fix <em>Download from java.sun.com</em> installation method for JDK for downloads requiring an Oracle login.
```

Comment:
I would explicitly mention that it is a change in theOracle website impacting previous versions. Once it's in LTS, maybe it makes sense to spin an upgrade blog post